error.go

```diff
@@ -0,0 +1,5 @@
+package tprstorage
+
+import "github.com/giantswarm/microstorage"
+
+var invalidConfigError = microstorage.InvalidConfigError
```

Comment:
> which is wrong in case an error originates in the tprstorage package.

You're right. Then we probably should create errors with `fmt.Errorf` since it's stacktrace of that file is just a noise.